frontend/src/search/SearchResultEntryGroup.ts

```diff
@@ -0,0 +1,17 @@
+import { SearchResultEntry } from './SearchResultEntry';
+
+/**
+ * Represents a group of option elements inside the dropdown of the autocomplete search.
+ * The groups are separated by the type of search results (node, edge, node-type, edge-type)
+ */
+
+export interface SearchResultEntryGroup {
+  /**
+   * group label that is visible to the user
+   */
+  label: string,
```

Comment:
```suggestion
  label: string;
```

Use the linter (`yarn lint --fix` might even do it)